src/main/java/com/rb/nonbiz/math/stats/MutableNormalDistributionGenerator.java

```diff
@@ -30,12 +33,12 @@ private MutableNormalDistributionGenerator(Normal normal, NormalDistribution nor
   }
 
   public static MutableNormalDistributionGenerator mutableNormalDistributionGenerator(
-      NormalDistribution normalDistribution, OptionalInt seed) {
+      NormalDistribution normalDistribution, Optional<RandomNumberGeneratorSeed> randomNumberGeneratorSeed) {
```

Comment:
Now that RandomNumberGeneratorSeed exists, we might as well use it elsewhere, instead of just a raw int.